examples/target/scala-2.13/examples-opt.js

```diff
@@ -0,0 +1,1214 @@
+(function(){'use strict';
```

Comment:
Can we drop this file?

Reply:
It's used for https://olivierblanvillain.github.io/monadic-html/examples/

Reply:
Better keep it then.